setup.py

```diff
@@ -25,7 +25,15 @@
     'pytest-random-order',
 ]
 
-all_extra = api + plot + develop
+jupyter = [
+    'jupyter',
+    'nbstripout',
+    'ipykernel',
+    'isort',
+    'yapf',
```

Comment:
i don't think `yapf` (or any other formatter for that matter) is loaded to jupyter automatically (you'll need a plugin for that) ... so i don't think this should be installed with jupyter.
same with `isort` ..

Reply:
Sure. I deleted them. I use them with nb extensions, but the user can figure that out if they choose to enable any extensions.